### PR TITLE
Migrate core/css.js to goog.module syntax

### PR DESCRIPTION
**Pull request by moniika**
## The basics

- [x] I branched from `goog_module`
- [x] My pull request is against `goog_module`
- [x] My code follows the [style guide](
      https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] My code is presented in the form suggested in the [module
      conversion guide](https://github.com/google/blockly/issues/5026)
- [x] I have run `npm test` locally already.

### Resolves

Part of #5026

### Proposed Changes

Converts `core/css.js` to `goog.module` syntax.

### CODE REVIEW

File: core/css.js [hunk 1]
```diff
@@ -14,28 +14,29 @@
...
+goog.module.declareLegacyNamespace();
 
 
 /**
  * Has CSS already been injected?
  * @type {boolean}
  * @private
  */
-Blockly.Css.injected_ = false;
+let injected_ = false;
```
Comment (gonfunko) [inline core/css.js:26]: I was under the impression that we also wanted to remove underscores (and possibly the @private? A bit unclear if we've decided on that yet)

File: core/css.js [hunk 2]
```diff
@@ -78,478 +78,273 @@ const inject = function(hasCss, pathToMedia) {
  */
 const CONTENT = [
```
Comment (gonfunko) [inline core/css.js:79]: I understand why clang-format did this, but ooph...possibly we want to avoid it here?

File: core/css.js (hunk 1 above)
Comment (moniika) [reply core/css.js]: Fixed.

File: core/css.js (hunk 2 above)
Comment (moniika) [reply core/css.js:79]: Yikes. I'm reverting that commit and I'll see if there's a better way to do this.
I'm thinking either to disable clang for these lines in some way or with a template string.

**Review by gonfunko — APPROVED** [review]

**Pull request merged**